pagopa/io-app
# chore: [IOBP-166] Add `Alert` feedback into expired payment methods detail

## ⚠️ This PR depends on #4867 ⚠️

## Short description
This PR adds an Alert message into expired payment methods detail.

## Design reference
[FIGMA](https://www.figma.com/file/f7jOjE3RokprMfb7c9NCVQ/Dettaglio-metodo-di-pagamento?type=design&node-id=1205-19503&mode=design&t=FoEVzT6goyKKIWNe-4)

## List of changes proposed in this pull request
- Added into `PaymentMethodFeatures` component a check to verify if the provided payment method is expired
    - If it is expired, it shows an `Alert` component with the expiration message inside;

## How to test
- Edit into `io-dev-api-server` the expiration date of a mocked payment method (into `wallet_v2.ts` file, edit at row `254` with `const ed = faker.date.past(10)`;
- Go into a credit card detail page and you should see an error Alert message

## Preview
<img src="https://github.com/pagopa/io-app/assets/34343582/f34ab0f6-6064-4010-a68a-181123399357" width="250" />
